addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -1294,4 +1294,103 @@ describe('studyService', () => {
       expect(retVal).toEqual(false);
     });
   });
+
+  describe('updatePermissions', () => {
+    it('should throw error if My Study but not migration request', async () => {
```

Comment:
that isn't so clear it is probably clearer to say: 
```suggestion
    it('should throw error if My Study and usersToRemove is not *', async () => {
```

Reply:
I am going to combine both messages, as they *should* be synonymous with each other because the only time `usersToRemove` should be `*` is when it is a migration request.